index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="style.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <header class="main-layout">
+      <div class="header-bar">
+        <a href="/">
+          <img
+            class="title"
+            src="img/img/Logo/Property 1=md.png"
+            alt="이미지"
+          />
+        </a>
+        <div class="login">
+          <a href="/login">로그인</a>
+        </div>
+      </div>
+    </header>
+    <section class="section-look main-layout">
+      <div class="section-look-contents">
+        <div class="section-look-content">
+          <h2>일상의 모든 물건을<br />거래해 보세요</h2>
+          <div class="look-button">
+            <a href="/items">구경하러 가기</a>
+          </div>
+        </div>
+        <img src="img/img/Landing page/Img_home_top.png" alt="거래 구경하기" />
+      </div>
+    </section>
+    <section class="section-items main-layout">
+      <div class="section-item">
+        <img src="img/img/Landing page/Img_home_01.png" alt="인기상품 확인" />
+        <div class="item-text">
+          <p class="item-hot">Hot item</p>
+          <h2>인기 상품을<br />확인해 보세요</h2>
+          <p class="item-description">
+            가장 HOT한 중고거래 물품을<br />판다 마켓에서 확인해 보세요
+          </p>
+        </div>
+      </div>
+    </section>
+    <section class="section-items main-layout">
+      <div class="section-item-reverse">
+        <div class="item-text item-text-reverse">
+          <p class="item-hot">Search</p>
+          <h2>구매를 원하는<br />상품을 검색하세요</h2>
+          <p class="item-description">
+            구매하고 싶은 물품은 검색해서<br />쉽게 찾아보세요
+          </p>
+        </div>
+        <img src="img/img/Landing page/Img_home_02.png" alt="관심상품 검색" />
+      </div>
+    </section>
+    <section class="section-items main-layout section-items-last">
+      <div class="section-item">
+        <img src="img/img/Landing page/Img_home_03.png" alt="판매상품 등록" />
+        <div class="item-text">
+          <p class="item-hot">Register</p>
+          <h2>판매를 원하는<br />상품을 등록하세요</h2>
+          <p class="item-description">
+            가장 HOT한 중고거래 물품을<br />판다 마켓에서 확인해 보세요
+          </p>
+        </div>
+      </div>
+    </section>
+    <section class="banner main-layout">
+      <div class="banner-contents">
+        <div class="banner-content">
+          <h2>믿을 수 있는<br />판다마켓 중고 거래</h2>
+        </div>
+        <img
+          src="img/img/Landing page/Img_home_bottom.png"
+          alt="광고배너이미지"
+        />
+      </div>
+    </section>
+    <footer class="footer-contents main-layout">
+      <div class="footer-content">
+        <div id="footer-license">©codeit - 2024</div>
+        <div class="footer-notice">
+          <a href="/privacy">Privacy Policy</a>
+          <a href="/faq">FAQ</a>
+        </div>
+        <div class="footer-sns">
+          <a href="https://www.facebook.com/" target="_blank">
+            <img src="img/icon/ic_facebook.png" alt="페이스북" />
+          </a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul class="footer-sns">
          <li>
            <a href="https://www.facebook.com/" target="_blank">
              <img src="img/icon/ic_facebook.png" alt="페이스북" />
            </a>
          </li>
```